internal/nodes/dequeue_ingestion.go

```diff
@@ -149,7 +149,7 @@ func DequeueIngestion(key string, fullsync bool) {
 	}
 
 	// Push Services from InfraSetting updates. Valid for annotation based approach.
-	if objType == lib.AviInfraSetting && !lib.UseServicesAPI() {
+	if objType == lib.AviInfraSetting && !lib.UseServicesAPI() && !lib.IsWCP() {
```

Comment:
how L4service updates, when Aviinfrasetting is updated , will be handled in wcp scenario?

Reply:
WCP works with Gateways, not L4 services